docs: docs for configuring system notifications (#84)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/setting-up/notifications.md

<img class="screenshot" alt="Set Message" src="{{docs_base_url}}/assets/img/setup/notifications/slack_notification_4.png">

---

## 3. System Notifications

> In **Version 12** we introduced System notifications for **assignments**, **mentions**, **documents shared**, and **energy points**. These notifications show up on the notifications dropdown on the top right of the navigation bar.

Reviewer: ```suggestion
In **Version 12** we introduced System notifications for **Assignments**, **mentions**, **documents shared**, and **Energy Points**. These notifications show up in the notifications dropdown on the the navigation bar's top right corner.
```

In **Version 13** we have introduced an additional channel to send alerts - **System Notifications**:

<img class="screenshot" alt="Notifications Dropdown" src="{{docs_base_url}}/assets/img/setup/notifications/system-notifications-channel.png">

Choosing this channel will send a system notification when a notification is triggered, instead of an Email or a Slack notification.

<img class="screenshot" height=400 alt="Notifications Dropdown" src="{{docs_base_url}}/assets/img/setup/notifications/system-notification.png">

Clicking on the notification routes to the **Notification Log** document which contains the configured subject, message as well as the attached file, if Attach Print is set:

<img class="screenshot" alt="Notifications Dropdown" src="{{docs_base_url}}/assets/img/setup/notifications/notification-log.png">

> In case Email/Slack alerts as well as System Notifications are required, the main channel can be set as Email or Slack and this option can be checked:

Reviewer: ```suggestion
If Email/Slack alerts and System Notifications both are required, the main channel can be set as Email or Slack and this option can be checked:
```
